### PR TITLE
105 create preview

### PR DESCRIPTION
**Pull request by T-z**

### CODE REVIEW

**Review by Filipe-Inacio — APPROVED** [review]

File: .project [hunk 1]
```diff
@@ -5,7 +5,24 @@
...
+			<id>1665486628338</id>
+			<name></name>
+			<type>30</type>
+			<matcher>
+				<id>org.eclipse.core.resources.regexFilterMatcher</id>
+				<arguments>node_modules|\.git|__CREATED_BY_JAVA_LANGUAGE_SERVER__</arguments>
+			</matcher>
+		</filter>
+	</filteredResources>
 </projectDescription>
```
Comment (asiday) [inline .project:28]: Should these proporties be added to the project?

File: am-ui/webcomponents/create-ad-webcomponent/package-lock.json [hunk 2]
```diff
@@ -25377,15 +25327,6 @@
         }
       }
     },
-    "swiper": {
-      "version": "8.4.2",
```
Comment (asiday) [inline am-ui/webcomponents/create-ad-webcomponent/package-lock.json:25381]: What is the differernce with the library "node_modules/swiper"?

File: am-ui/webcomponents/create-ad-webcomponent/src/components/CreateAdFields.vue [hunk 3]
```diff
@@ -402,12 +404,16 @@ onMounted(() => {
   props.fieldsToShow.forEach((field) => {
     // checkbox default value shoud be false
     if (field.field_type_definition_id === 8 || field.field_type_definition_id === 16) {
-      fieldValues.value[field.field_id] = false;
-    } else {
-      fieldValues.value[field.field_id] = '';
+      fieldValues.value[field.field_id] = field.value ?? false;
+    } else if (field.field_type_definition_id === 6){
+      //selectbox
```
Comment (asiday) [inline am-ui/webcomponents/create-ad-webcomponent/src/components/CreateAdFields.vue:409]: Could you please add the meaning of constatns?

File: infrastructure/start_all_local/clean_startup.sh [hunk 4]
```diff
@@ -4,4 +4,4 @@ docker image prune -f
 docker image rm start_all_local_marketplace-service:latest start_all_local_admin-ui:latest start_all_local_auth-service:latest start_all_local_client-ui:latest
 docker volume rm am_db --force
 docker volume rm keycloak_db --force
-docker-compose up
\ No newline at end of file
+docker-compose up -d
```
Comment (asiday) [inline infrastructure/start_all_local/clean_startup.sh:7]: May be leave "docker-compose up" in non-detached mode?

**Review by asiday — APPROVED** [review]

File: infrastructure/start_all_local/clean_startup.sh (hunk 4 above)
Comment (T-z) [reply infrastructure/start_all_local/clean_startup.sh:7]: the paramater -d is optional. We dont neccessary need it. A matter of preference.

**Pull request merged**